docs/release-notes/3.0.0_Beta.md

```diff
@@ -51,3 +51,8 @@ This release introduces major breaking API changes:
 - Low-level Inventory API sections have been removed from documentation in favor of the Management API for inventory-related tasks.
 
 For detailed information about all these API changes and new endpoints, see the [Release Notes in the PMM API Documentation](https://percona-pmm.readme.io/v3/reference/release-notes/3.0.0_Beta). [Release Notes in the PMM API Documentation](https://percona-pmm.readme.io/v3/reference/release-notes/3.0.0_Beta).
+
+
+## Components upgrade
+
+Grafana has been upgraded to version 11.1.8.
```

Comment:
```suggestion
PMM now integrates Grafana 11.1.8, which delivers enhancements alongside all the advancements introduced since the previous Grafana 10.4 integration.

For the full list of  Grafana changes included with this update, see [Grafana’s 10.1.8 changelog](https://community.grafana.com/t/changelog-updates-in-grafana-11-1-8/134843) and [Grafana release highlights](https://grafana.com/docs/grafana/latest/whatsnew/).```

Reply:
@catalinaadam 

Similary to what was mentioned regarding the tabs issue in alpha. The latest publicly available grafana version in pmm is 9.2.20 in PMM 2.

Also there's a typo: [Grafana's 1**1**.1.8].

Reply:
@matejkubinec, thnks - I've updated my suggestion. @YashSartanpara1, can I commit the suggestion and merge the PR?

Reply:
Sure Please.